ncdump/CMakeLists.txt

```diff
@@ -242,7 +243,7 @@ endif()
     endif()
 
     SET_TESTS_PROPERTIES(ncdump_tst_nccopy3 PROPERTIES DEPENDS
-        "ncdump_tst_calendars;ncdump_run_utf8_tests;ncdump_tst_output;ncdump_tst_64bit;ncdump_run_tests;ncdump_tst_lengths")
+        "ncdump_tst_calendars;ncdump_tst_calendars_nc4;ncdump_run_utf8_tests;ncdump_tst_output;ncdump_tst_64bit;ncdump_run_tests;ncdump_tst_lengths")
```

Comment:
I'm going to remove this addition. I misunderstood and now am thinking that the new `ncdump_tst_calendars_nc4` test does not belong here since the `ncdump_tst_nccopy3` test does not depend on it.